package.json

```diff
@@ -258,6 +258,7 @@
     "minimist": "^1.2.6",
     "moment:>2.0.0 <3": ">=2.29.4",
     "node-forge": ">=1.3.0",
+    "nest-typed-config/class-validator": "0.14.1",
```

Comment:
<s>Is this used somewhere? I don't see any file in this diff using it, or in this `git grep` (unless we oddly need it as a peer dep):

```sh
git grep "nest-typed-config/class-validator" | cat

package.json:    "nest-typed-config/class-validator": "0.14.1",
```
</s>

Nevermind, I see it's in `resolutions` and not as a `dependency` or `devDependency`.